src/MCPServer/lib/server/shared_dirs.py

```diff
@@ -261,16 +278,8 @@ def create():
         "sharedMicroServiceTasksConfigs/transcoder/defaultIcons",
         "SIPbackups",
         "tmp",
-        "watchedDirectories",
-        "watchedDirectories/activeTransfers",
-        "watchedDirectories/activeTransfers/baggitDirectory",
-        "watchedDirectories/activeTransfers/baggitZippedDirectory",
-        "watchedDirectories/activeTransfers/dataverseTransfer",
-        "watchedDirectories/activeTransfers/Dspace",
-        "watchedDirectories/activeTransfers/maildir",
-        "watchedDirectories/activeTransfers/standardTransfer",
-        "watchedDirectories/activeTransfers/TRIM",
-        "watchedDirectories/activeTransfers/zippedDirectory",
+        WATCHED_DIRS,
+        ACTIVE_TRANSFERS_DIR,
```

Comment:
Were there to be any doubt about this, (we had tests to prove the data which we've created to so rely on on AM of present), this code would fail immediately on start-up, and then watched directories after that would simply not work for any transfer too. `black-box` tests cover AIP generation. We'd know pretty quick.